ceph.spec.in

```diff
@@ -813,18 +816,42 @@ export CPPFLAGS="$java_inc"
 export CFLAGS="$RPM_OPT_FLAGS"
 export CXXFLAGS="$RPM_OPT_FLAGS"
 
-env | sort
-
+# Parallel build settings ...
+# unlimit _smp_mflags in system macro
+%global _smp_ncpus_max 0
```

Comment:
Why do we override this? I know we did that before but if we are changing things I would be in favour of removing this line. Why don't we let the users configure their own _smp_ncpus_max? It should default to 0 anyway.